extensions/rich_text_components/Svgdiagram/protractor.js

```diff
@@ -59,7 +59,7 @@ const SVGTAGS = {
     '1999/xlink" version="1.1" width="494" height="338" viewBox="0 0 494 338' +
     '"><desc>Created with Fabric.js 4.1.0</desc><defs></defs><rect x="0" y="' +
     '0" width="100%" height="100%" fill="rgba(0,0,0,0)"/><g transform="matri' +
-    'x(1 0 0 1 50 50)" style=""><text font-family="helvetica" font-size="18"' +
+    'x(1 0 0 1 50 50)" style=""><text font-family="Helvetica" font-size="18"' +
```

Comment:
Note: Since these changes should not affect e2e tests, you do not need to re-run the tests 5 consecutive times.